core/common/src/main/java/alluxio/security/authorization/AccessControlList.java

```diff
@@ -425,10 +425,19 @@ private void setOtherEntry(AclEntry entry) {
    * @return {@link AccessControlList}
    */
   public static AccessControlList fromProtoBuf(File.AccessControlList acl) {
-    AccessControlList ret = new AccessControlList();
+    AccessControlList ret;
+    if (acl.hasIsDefault() && acl.getIsDefault()) {
+      ret = new DefaultAccessControlList();
+    } else {
+      ret = new AccessControlList();
+    }
     ret.setOwningUser(acl.getOwningUser());
     ret.setOwningGroup(acl.getOwningGroup());
 
+    if (acl.getIsEmpty()) {
```

Comment:
Is a normal `AccessControlList` able to be empty?

Reply:
normal `AccessControlList` is always non-empty.